R/checkMissings.R

```diff
@@ -79,6 +90,14 @@ checkMissingsByValues <- function(GADSdat, missingValues = -50:-99, addMissingCo
 #'@export
 checkMissingsByValues.GADSdat <- function(GADSdat, missingValues = -50:-99, addMissingCode = TRUE) {
   check_GADSdat(GADSdat)
+  if(!is.numeric(missingValues) || length(missingValues) == 0) {
+    stop("'missingValues' needs to be a numeric vector of at least length 1.")
+  }

```

Comment:
Nit: Ah, maybe, the type abstraction would not work here (see my draft in `check_functions()`). The routine is quite similar to the check routine for the characters, but the length condition is different...